apps/portals/src/types/portal-util-types.ts

```diff
@@ -59,4 +59,6 @@ export type DetailsPageTabProps = DetailsPageContent & {
   iconName?: string
   cssClass?: string
   toolTip?: string
+  /** Given this column name, if the assocated value in this column is not set, then this tab will be hidden */
+  hideIfColumnValueUndefined?: string
```

Comment:
The actual JSON values Synapse returns for a row are string or null, so l think this name would be a bit more accurate

```suggestion
  /** Given this column name, if the assocated value in this column is not set, then this tab will be hidden */
  hideIfColumnValueNull?: string
```